### PR TITLE
Bubble sort feature

### PR DESCRIPTION
**Pull request by GraceOyiza**
- Add a bubble_sort method.
- Add a bubble_sort_by method.

### CODE REVIEW

**Comment by GraceOyiza** [comment]
- Linters error fixed

File: README.md [hunk 1]
```diff
@@ -2,58 +2,37 @@
 
 # Project Name
```
Comment (krys2fa) [inline README.md]: - [ ] Please replace this with the actual name of the project or remove it completely.

File: README.md [hunk 2]
```diff
@@ -2,58 +2,37 @@
...
 
-> One paragraph statement about the project.
+One paragraph statement about the project.
```
Comment (krys2fa) [inline README.md]: - [ ] Please replace this with the actual statement on the project or remove it completely.

File: README.md [hunk 3]
```diff
@@ -2,58 +2,37 @@
...
 
-![screenshot](./app_screenshot.png)
+- Advanced Building Blocks - Implementation of the bubble sort algorithm.
 
 Additional description about the project and its features.
```
Comment (krys2fa) [inline README.md]: - [ ] Please replace this with the actual description or remove it completely.

**Review by krys2fa — CHANGES_REQUESTED** [review]
## Awesome work @GraceOyiza and @mohapakram 👏 
Your project displays good use of Github Flow workflow and has descriptive commit messages. 💪 

### Status: Changes Required ♻️ 
It is important that your repo is as professional as possible. As a result, kindly make the changes in your README file as stated below

Please request for another review after implementing the changes.

Happy Coding 🖥️  😃 

###### You can contact me on slack (Chris Amanor) for any clarification.

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi @GraceOyiza and @mohapakram, great job fixing the comments made by the previous TSE :+1: The README is fixed with the necessary text. I hereby approve of the project, congratulations :fireworks: 

Go ahead and merge to master :slightly_smiling_face: 
_Happy coding_ :woman_technologist: :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**